cns/restserver/internalapi_test.go

```diff
@@ -475,6 +485,188 @@ func TestReconcileNCWithExistingState(t *testing.T) {
 	validateNCStateAfterReconcile(t, req, expectedNcCount+1, expectedAssignedPods, nil)
 }
 
+func TestReconcileCNSIPAMWithDualStackPods(t *testing.T) {
```

Comment:
can we add test case for single stack v4 and with system pods IPs?

Reply:
i know some of the code/comments say "system pods" but that doesn't actually mean anything. do you mean hostnetwork pods?

Reply:
yes you are right. I meant hostnetworking pods